spec/controllers/admin/resets_controller_spec.rb

```diff
@@ -13,12 +13,12 @@
 
   describe 'POST #create' do
     it 'redirects to admin accounts page' do
-      expect_any_instance_of(User).to receive(:send_reset_password_instructions) do |value|
-        expect(value.account_id).to eq account.id
-      end
-
-      post :create, params: { account_id: account.id }
+      expect do
+        post :create, params: { account_id: account.id }
+      end.to change(Devise.mailer.deliveries, :size).by(2)
 
+      expect(Devise.mailer.deliveries.first.to).to include(account.user.email)
+      expect(Devise.mailer.deliveries.last.to).to include(account.user.email)
```

Comment:
Can this be made more specific on the mail type being sent?

Reply:
We can't get the class but yes we can check for more detail. Added a commit which checks both the `to` and the `subject` of both of the mails sent here, which is probably enough.